src/main/java/com/google/cloud/tools/maven/CloudSdkChecker.java

```diff
@@ -16,11 +16,49 @@
 
 package com.google.cloud.tools.maven;
 
-import org.apache.maven.plugin.logging.Log;
+import com.google.cloud.tools.appengine.cloudsdk.AppEngineJavaComponentsNotInstalledException;
+import com.google.cloud.tools.appengine.cloudsdk.CloudSdk;
+import com.google.cloud.tools.appengine.cloudsdk.CloudSdkNotFoundException;
+import com.google.cloud.tools.appengine.cloudsdk.CloudSdkOutOfDateException;
+import com.google.cloud.tools.appengine.cloudsdk.CloudSdkVersionFileException;
+import com.google.common.annotations.VisibleForTesting;
 
 public class CloudSdkChecker {
 
-  public void checkCloudSdk(Log log) {
-    log.warn("Cloud SDK checker not yet implemented.");
+  private String version;
+
+  CloudSdkChecker(String version) {
+    this.version = version;
+  }
+
+  @VisibleForTesting
+  public void setVersion(String version) {
```

Comment:
I wonder if we need this, give the constructor takes the same inputs?

Reply:
True. I was testing multiple versions and was injecting mocks, so I didn't want to re-instantiate the tested instance. But it looks like things have changed enough that I can remove this without affecting anything.